Makefile

```diff
@@ -94,10 +94,10 @@ misspell:
 .PHONY: reload_docs
 reload_docs: generator docs
 
-# Download and setup tooling dependencies.
-.PHONY: setup
-setup: ve
-	cd scripts && $(FORCE_GO_MODULES) go mod download
```

Comment:
Why are you deleting the `setup` target?

Reply:
I think cut instead of copied. Fixed now.